frontend/app/model/Schema.scala

```diff
@@ -39,6 +48,7 @@ case class EventSchema(
  eventStatus:String,
  location: Option[LocationSchema],
  offers: Option[OfferSchema],
+ organizer:Option[OrganizerSchema],
```

Comment:
Just another formatting/readability point, but there should be a space between the `organizer:` and `Option[OrganizerSchema]` on this line.

Reply:
I'd also add a space in the same way after the `eventAttendanceMode` and `eventStatus` keys.